tox.ini

```diff
@@ -25,7 +29,8 @@ deps =
 deps =
     # sphinx
     Sphinx==4.2.0
-    django-dbdiff
+    ; django-dbdiff
+    git+https://github.com/pfouque/django-dbdiff.git@fix312#egg=django-dbdiff
```

Comment:
@marianoeramirez you're now hooked to my fork of django-dbdiff ... :/

Reply:
Yes, but we will have to wait a long time before them review and integrate the PR. Once that happen I will update the project.

Reply:
Ok, I'm fine with that, I wanted to be sure you noticed! ;) 

In the meantime, I started a branch to remove this dependency, 
and use load_fixtures/dump_fixtures instead + a backport of the assertNoDiff statement
It was promising, and only the command was missing tests.

Do you want me to publish the draft?

Reply:
https://github.com/yourlabs/django-cities-light/pull/288